src/components/partnership/partnership.service.ts

```diff
@@ -164,6 +164,10 @@ export class PartnershipService {
         ...securedProps.types,
         value: securedProps.types.value ?? [],
       },
+      partner: {
+        ...securedProps.partner,
+        value: securedProps.partner.canRead ? dto.partner : undefined,
+      },
```

Comment:
This looks like the exact logic that secureProperties does. Why was this needed?

Reply:
Oh. I thought that that was a placeholder for the id until it got called and filled in in the resolver. I overthought it. I'll take it out and recommit.